products/compute/api.yaml

```diff
@@ -1158,6 +1159,58 @@ objects:
               "Cache-Control: public, max-age=[TTL]" header, regardless of any
               existing Cache-Control header. The actual headers served in
               responses will not be altered.
+          - !ruby/object:Api::Type::Integer
+            name: 'defaultTtl'
+            min_version: beta
+            description: |
+              Specifies the default TTL for cached content served by this origin for responses 
+              that do not have an existing valid TTL (max-age or s-max-age).
+          - !ruby/object:Api::Type::Integer
+            name: 'maxTtl'
+            min_version: beta
+            description: |
+              Specifies the maximum allowed TTL for cached content served by this origin.
+          - !ruby/object:Api::Type::Integer
+            name: 'clientTtl'
+            min_version: beta
+            description: |
+              Specifies the maximum allowed TTL for cached content served by this origin.
+          - !ruby/object:Api::Type::Boolean
+            name: 'negativeCaching'
+            min_version: beta
+            description: |
+              Negative caching allows per-status code TTLs to be set, in order to apply fine-grained caching for common errors or redirects.
+          - !ruby/object:Api::Type::Array
+            name: 'negativeCachingPolicy'
+            min_version: beta
+            description: |
+              Sets a cache TTL for the specified HTTP status code. negativeCaching must be enabled to configure negativeCachingPolicy.
+              Omitting the policy and leaving negativeCaching enabled will use Cloud CDN's default cache TTLs.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::Integer
+                  name: 'code'
+                  min_version: beta
+                  description: |
+                    The HTTP status code to define a TTL against. Only HTTP status codes 300, 301, 308, 404, 405, 410, 421, 451 and 501
+                    can be specified as values, and you cannot specify a status code more than once.
+                - !ruby/object:Api::Type::Integer
+                  name: 'ttl'
+                  min_version: beta
+                  description: |
+                    The TTL (in seconds) for which to cache responses with the corresponding status code. The maximum allowed value is 1800s
+                    (30 minutes), noting that infrequently accessed objects may be evicted from the cache before the defined TTL.
+          - !ruby/object:Api::Type::String
```

Comment:
It looks like this is an enum in the API - should it be an enum here too?

Reply:
Yep, definitely, let me change that...